packages/graphql/tests/tck/issues/3394.test.ts

```diff
@@ -59,7 +59,7 @@ describe("https://github.com/neo4j/graphql/issues/3394", () => {
             "MATCH (this:Product)
             WITH *
             ORDER BY this.fg_item DESC
-            RETURN this { .description, .fg_item, id: this.fg_item_id, partNumber: this.fg_item } AS this"
```

Comment:
Note that these tests were changed already in translation-layer

The new Cypher is closer to the original one in 4.0.0

Reply:
That's great!